src/include/txn.i

```diff
@@ -1294,12 +1313,8 @@ static inline void
 __wt_txn_cursor_op(WT_SESSION_IMPL *session)
```

Comment:
I'm a bit concerned about this.

We call this directly in some cursor logic regardless of the `cbt` flags. I believe that we're still acquiring snapshots in some cursor operations (even before this change).